steps/create_image.cwl

```diff
@@ -97,18 +95,10 @@ arguments:
     valueFrom: $(inputs.numcpu)
   - prefix: '-mem'
     valueFrom: $(inputs.mem_pct)
-  - prefix: ''
-    separate: false
-    valueFrom: '-no-update-model-required'
-  - prefix: ''
-    separate: false
-    valueFrom: '-reorder'
-  - prefix: ''
-    separate: false
-    valueFrom: '-fit-beam'
-  - prefix: ''
-    separate: false
-    valueFrom: '-join-channels'
+  - valueFrom: '-no-update-model-required'
+  - valueFrom: '-reorder'
```

Comment:
If only `valueFrom` is used then you can omit it :-)

``` yaml
 - -no-update-model-required
 - -reorder
```

Reply:
Check!